widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.kt

```diff
@@ -173,10 +173,29 @@ object GliaWidgets {
                         onComplete.onComplete()
                     } else {
                         Logger.i(TAG, "Glia Widgets SDK initialization failed")
-                        val invalidInputError = GliaWidgetsException(
-                            "Failed to initialise Glia Widgets SDK. Please check credentials.",
-                            GliaWidgetsException.Cause.INVALID_INPUT
-                        )
+                        val invalidInputError = when (exception.cause) {
+                            GliaException.Cause.NETWORK_TIMEOUT -> {
+                                GliaWidgetsException(
+                                    "Network timeout. Please check the Internet connection.",
+                                    GliaWidgetsException.Cause.NETWORK_TIMEOUT
+                                )
+                            }
+
+                            GliaException.Cause.INVALID_INPUT,
+                            GliaException.Cause.FORBIDDEN -> {
+                                GliaWidgetsException(
+                                    "Failed to initialise Glia Widgets SDK. Please check credentials.",
+                                    GliaWidgetsException.Cause.INVALID_INPUT
+                                )
+                            }
+
+                            else -> {
+                                GliaWidgetsException(
+                                    "Failed to initialise Glia Widgets SDK. Please check credentials.",
+                                    GliaWidgetsException.Cause.INTERNAL_ERROR
```

Comment:
I might be mistaken, but I think SDK should not throw `GliaWidgetsException.Cause.INTERNAL_ERROR` for internal errors. As mentioned in casse heigher it should be `GliaWidgetsException.Cause.INVALID_INPUT`. Maybe we could change this error message?
```suggestion
                                GliaWidgetsException(
                                    "Failed to initialise Glia Widgets SDK. Internal error, please check logs.",
                                    GliaWidgetsException.Cause.INTERNAL_ERROR
```

Reply:
Unfortunately, conditions above don't cover all credential cases.
Also, it is not a good idea to direct integrators to logs, because logs say nothing.